src/nvim/window.c

```diff
@@ -506,11 +506,23 @@ static void cmd_with_count(char *cmd, char_u *bufp, size_t bufsize,
  *
  * return FAIL for failure, OK otherwise
  */
-int win_split(int size, int flags)
+int win_split(int size, int flags) {
+    return win_split_with_tab_suc(size, flags, NULL);
+}
+
+int win_split_with_tab_suc(int size, int flags, bool *tab_suc)
 {
-  /* When the ":tab" modifier was used open a new tab page instead. */
-  if (may_open_tabpage() == OK)
+  if (tab_suc != NULL) {
+    *tab_suc = false;
+  }
+
+  // When the ":tab" modifier was used open a new tab page instead.
+  if (may_open_tabpage() == OK) {
+    if (tab_suc != NULL) {
+      *tab_suc = true;
```

Comment:
I had that before, but the result in that argument would be just the same as the return value (except for it being a bool instead of OK/FAIL integer)
Another solution would be to let ```win_split``` have a third return value ```OK_IN_NEW_TAB```, then we don't even need a ```tab_suc``` argument altogether. What do you think is best?